src/peirce_api_calls.ts

```diff
@@ -44,7 +48,173 @@ export interface PopulateAPIReponse {
     adata:PopulateAPIData[];
 }
 
+// merges all terms and spaces from a file into a singular list of type (Term : Space)[]
+// technically this could be done in linear time if we notice that this is too slow, but I don't think that there
+// are any issues with non-extreme use cases
+const mergeSpaces = (): models.Space[] =>{
+    let geom3d = peircedb.getGeom3DSpaces();
+    let geom1d = peircedb.getGeom1DSpaces();
+    let time = peircedb.getTimeSpaces();
+    let res : models.Space[] = [];
+
+    console.log(geom3d);
+    console.log(geom1d);
+    console.log(time);
+    geom1d.forEach(element =>{
+        if (element.fileName == getActivePeirceFile()) {
```

Comment:
why is this here?

Reply:
This is a file guard that I added, and must have forgotten to remove them after I updated how the calls to getPeirceDB() work. In theory, we can remove this (and all other file guards I have forgotten to remove).